docker/releases/build_releases.sh

```diff
@@ -53,50 +57,65 @@ while [ "$1" != "" ]; do
     shift
 done
 
-if [[ ! "$NETWORK" =~ ^mainnet$|^testnet$|^betanet$ ]]
```

Comment:
This was confusing split logic. Combined these into a simpler case statement.

Reply:
Appreciate the improved clarity.